README.md

```diff
@@ -1,3 +1,5 @@
 # test-01
 
 jhhjkjhjkhkjs
+
+;wjkjewhkjeh
```

Comment:
Please elaborate on the purpose of these changes in the README. What does "jhhjkjhjkhkjs;wjkjewhkjeh" signify?